db/migrate/20180226170900_populate_team_category_roles.rb

```diff
@@ -1,3 +1,21 @@
+class BusinessUnit
```

Comment:
Don't we need to inherit from `ActiveRecord::Base` here? Has this been tested?

Reply:
Also, why did this get renamed, is that the right thing to do? I expected the date in the name to stay the same.

Reply:
yes, it's been tested.  For some reason, later migrations are already on production (and dev and demo), so this doesn't get run.  I thought it would do missing ones, but apparently not.